test/cypress/integration/group-proposals.spec.js

```diff
@@ -319,51 +330,61 @@ describe('Proposals - Add members', () => {
       .should('have.length', 4)
   })
 
-  it('It should receive non monetary contribution', () => {
+  it('User2 pledges $100 and sees the non-contributions they are receiving.', () => {
     cy.giSwitchUser(`user2-${userId}`)
     updateIncome(100, false)
 
+    cy.get('.receiving .c-contribution-item:first-child')
+      .should('contain', 'French classes by Margarida')
+
     cy.get('.receiving .c-contribution-list')
       .should('have.length', 4)
+  })
 
+  it('User 2 also give non monetary contribution', () => {
     addNonMonetaryContribution('Korean classes')
     addNonMonetaryContribution('French classes')
 
     cy.get('.giving .c-contribution-list')
       .should('have.length', 3)
   })
 
-  it('It should give monetary contribution', () => {
+  it('User 3 pledges $100 and sees who they are pledging to - $50 to user1 (Margarida)', () => {
     cy.giSwitchUser(`user3-${userId}`)
     updateIncome(100, false)
+
     cy.get('.giving .c-contribution-item:first-child')
       .should('contain', '$50 to Margarida')
   })
 
-  it('It should give to many contribution', () => {
+  it('User 4 and user 2 change their pledges to $500 each. User1 sees the receiving contributions from 3 members.', () => {
     cy.giSwitchUser(`user4-${userId}`)
     updateIncome(500, false)
     addNonMonetaryContribution('Korean classes')
+
     cy.giSwitchUser(`user2-${userId}`)
     updateIncome(500, false)
+
     cy.giSwitchUser(`user1-${userId}`)
     cy.getByDT('contributionsLink').click()
     cy.get('.receiving .c-contribution-item:first-child')
       .should('contain', '$100 from 3 members')
   })
 
-  it('It should give to many contribution', () => {
+  it('After lower the income of user 4 and 2, user 3 should give to 3 persons', () => {
     cy.giSwitchUser(`user4-${userId}`)
     updateIncome(10, true)
+
     cy.giSwitchUser(`user2-${userId}`)
     updateIncome(10, true)
+
     cy.giSwitchUser(`user3-${userId}`)
     cy.getByDT('contributionsLink').click()
     cy.get('.giving .c-contribution-item:first-child')
       .should('contain', 'A total of $100 to 3 members')
   })
 
-  it('It should receive korean classes from many people', () => {
+  it('User 3 should receive the same non monetary contribution from 3 members', () => {
```

Comment:
@pieer I'm sorry to be this picky but u still have the same typos across different descriptions. user3 without spaces. incorrect verbs on 3rd person (receives, changes, gives, cancels, adds, etc...)

Reply:
IT's actually not user2 but user2234532 or user2345423 so... not sure what's the point of your requests.. but sure I can change whatever you want so we can close this

Reply:
the ids change for reasons we all know, but the username itself (user1, user2, user3, etc...) are constant. Explicit saying the username in a description, help us to better understand they actions and the test flow.